tools/building.py

```diff
@@ -551,6 +554,12 @@ def get_command_with_possible_response_file(cmd):
 # This function can be called either for a single file output listing ("llvm-nm a.o", or for
 # multiple files listing ("llvm-nm a.o b.o").
 def parse_llvm_nm_symbols(output):
+  # If response files are used in a call to llvm-nm, the output printed by llvm-nm has a
+  # quirk that it will contain double backslashes as directory separators on Windows. (it will
+  # not remove the escape characters when printing)
+  # Therefore canonicalize the output to single backslashes.
+  output = output.replace('\\\\', '\\')
```

Comment:
Just to be clear, if I remove this line then `test_llvm_nm_relative_paths_works_with_response_files` will fail on windows?

Reply:
yes